ModularTegustation/lc13_obj/_adventure_console/adventure_layout.dm

```diff
@@ -429,14 +559,68 @@
 	if(!value || !type)
 		return FALSE
 	switch(type)
+		//Health
 		if("HP")
-			if(virtual_integrity >= 100)
+			if(virtual_integrity >= max_integrity)
 				return FALSE
 			AdjustHP(value)
+		if("MAXHP")
+			if(value < max_integrity)
+				return FALSE
+			max_integrity = value
+
+		//Attack
 		if("DICE")
 			if(virtual_damage == value)
 				return FALSE
 			ChangeDice(value)
+
+		//Block
+		if("BLOCKDICE")
+			if(virtual_block == value)
+				return FALSE
+			ChangeBlockDice(value)
+
+		if("BLOCKHEAL")
+			if(value < block_heal)
+				return FALSE
+			block_heal = value
+
+		if("BLOCKMAX")
+			if(value < max_block_heal)
+				return FALSE
+			max_block_heal = value
+
+		//Running
+		if("RUNCHANCE")
+			if(value < run_chance)
+				return FALSE
+			run_chance = value
+
+		if("RUNHEAL")
+			if(value < run_healing)
+				return FALSE
+			run_healing = value
+
+		if("RUNGOLD")
+			if(value < run_gold)
+				return FALSE
+			run_gold = value
+
+		//Info
+		if("INFOCHANCE")
+			if(value < info_chance)
+				return FALSE
+			info_chance = value
+
+		if("INFOBONUS")
+			if(value < info_bonus)
+				return FALSE
+			info_bonus = value
+
+		//Extra
+		if("ABNOS")
+			all_abnos = TRUE
```

Comment:
```suggestion
			if(all_abnos)
				return FALSE
			all_abnos = TRUE
```
This is to prevent people buying extra content twice. I know its a bit silly but it prevents complaints.